### DISCUSSION TITLE
List all settings in databend home

### DISCUSSION THREAD

**Discussion by JasonLi-cn** [databendlabs/databend#4881] [Q&A] [answered]
Databend should list all settings in one page [https://databend.rs/doc/](url) as ck does [https://clickhouse.com/docs/en/operations/settings/settings/](url)

Because I want to know if I can change the parallelism of tasks when querying, and how.
I've tried "set max_threads=32", but it didn't work.

**Comment by PsiACE** [comment, marked as answer]
for settings:

https://databend.rs/doc/reference/sql/system-tables/system-settings

**Reply by JasonLi-cn** [reply]
thanks~
